frontend/src/component/menu/Header/Header.tsx

```diff
@@ -158,9 +156,7 @@ const Header: VFC = () => {
                 <StyledContainer>
                     <Tooltip title="Menu" arrow>
                         <IconButton
-                            sx={theme => ({
-                                color: theme.palette.text.tertiaryContrast,
-                            })}
+                            sx={{ color: 'black' }}
```

Comment:
```suggestion
                            sx={{ color: theme => theme.palette.text.primary }}
```
or `theme.palette.common.black`